src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/cdi/ManagedBeanConstructorQuickFix.java

```diff
@@ -17,4 +17,9 @@ public class ManagedBeanConstructorQuickFix extends InsertAnnotationMissingQuick
     public ManagedBeanConstructorQuickFix() {
         super("jakarta.inject.Inject");
     }
+
```

Comment:
Should update the copyright in this file like all the others.

Reply:
Updated the Copyrights